paasta_tools/deployd/workers.py

```diff
@@ -65,8 +67,13 @@ def run(self):
                                                                                soa_dir=marathon_tools.DEFAULT_SOA_DIR,
                                                                                marathon_config=self.marathon_config,
                                                                                marathon_apps=marathon_apps)
-            except Exception:
-                return_code, bounce_again_in_seconds = -1, 60
+            except Exception as e:
+                self.log.warning("deploy_marathon_service caused exception: {}".format(e))
+                return_code = -2
+            if return_code != 0:
+                failures += 1
+                bounce_again_in_seconds = exponential_back_off(failures, factor=60, base=2, max_time=6000)
```

Comment:
this seems quite high to me - can we make it configurable so we've got an easy way of experimenting to find the right values?